### PR TITLE
Increase count of nft

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2612** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2612" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2612?utm_source=stack-comment).

File: src/pages/api/ipfs/logo.page.ts [hunk 1]
```diff
@@ -15,42 +15,31 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
...
-        })
-      }
-      const file = files.file as formidable.File
-      const stream = fs.createReadStream(file.filepath)
-      let options
-      if (fields.pinataMetadata) {
-        const pinataMetadata = JSON.parse(fields.pinataMetadata as string)
-        options = {
-          pinataMetadata,
+    const result = await new Promise((resolve, reject) => {
```
Comment (wraeth-eth) [inline src/pages/api/ipfs/logo.page.ts:18]: Re-ordered some of the code here as I don't think it was working as intended

Comment (tomquirk) [reply src/pages/api/ipfs/logo.page.ts:18]: yep, you're correct 😂 was never actually returning anything cos the returns were in the `parse` callback

**Review by tomquirk — APPROVED** [review]

**Review by johnnyd-eth — APPROVED** [review]
LGTM

**Pull request merged**